### PR TITLE
feat(eLife): Add executable version download link

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

**Comment by discodavey** [comment]
@alex-ketch can you please add @nlisgo to this project so I can get him to review PR's

**Comment by alex-ketch** [comment]
@discodavey I've sent an invite to @nlisgo
Let me know if there's anything else :)

**Comment by discodavey** [comment]
@nlisgo Cannot add you as a reviewer so can you look at this and say if you are happy with the code?

File: src/themes/elife/lib/downloads.ts [hunk 1]
```diff
@@ -22,7 +22,17 @@ const deriveUrl = (type: string, id: string, title = ''): string => {
 const buildLinkToFiguresPdf = (url: string): void => {
   after(
     select('[data-is-download-pdf-link]')[0],
-    create('li', null, create('a', { href: url }, 'Figures PDF'))
+    create('li', null, create('a', { href: url }, 'Figures PDF')),
+    create(
+      'li',
+      null,
+      create('a', { href: url }, 'Executable version'),
```
Comment (nlisgo) [inline src/themes/elife/lib/downloads.ts:29]: We either need to change the function name from `buildLinkToFiguresPdf` or this belongs in a new function. If this is just placeholder for now then just use `#` as the href rather that the figures pdf url.

**Comment by nlisgo** [comment]
This is almost ready. I just want to review whether the era download will be introduced regardless of there being a figures PDF or not.

**Comment by discodavey** [comment]
@nlisgo Just a question. When I click the Executable Version link I get a blocked when on Firefox. This is from my localhost but would this cause issues crossdomain wise from the iframe or not?

Screenshot 2020-06-25 at 11 18 40

**Comment by nlisgo** [comment]
> would this cause issues crossdomain wise from the iframe or not?

Let's check when stencila hub is back up as the Mendeley link is similar. If it's an issue then we can log it and find a work around.

**Comment by nlisgo** [comment]
I'm leaving this in draft until we fix the issue with the figures pdf link: https://github.com/stencila/thema/pull/200

**Comment by nlisgo** [comment]
This can be simplified even further once https://github.com/stencila/thema/pull/205 is in

**Pull request merged**